frontend/src/app/(core)/layout.tsx

```diff
@@ -1,13 +1,23 @@
+'use client'
+
 import { Navbar, Footer } from '@sections'
 import React, { PropsWithChildren } from 'react'
+import {  SnackbarProvider } from 'notistack'
 
 const Layout = ({ children }: PropsWithChildren) => {
   return (
-    <>
+    <SnackbarProvider
+      maxSnack={3}
+      anchorOrigin={{
+        vertical: 'top',
+        horizontal: 'right',
```

Comment:
Can add `disableWindowBlurListener` prop here to disable snackbar exit on clicking the screen. Right now the snackbar hides as soon as we click anywhere.